src/chainparams.cpp

```diff
@@ -80,7 +80,7 @@ class CMainParams : public CChainParams {
         consensus.nPowTargetSpacing = 10 * 60;
         consensus.fPowAllowMinDifficultyBlocks = false;
         consensus.fPowNoRetargeting = false;
-        consensus.nRuleChangeActivationThreshold = 1916; // 95% of 2016
+        consensus.nRuleChangeActivationThreshold = 1815; // 90% of 2016
```

Comment:
Maybe add a note here to never change this unless all soft forks are burried.

Reply:
Can be fixed (in a follow-up refactor) with commit 2be730c31de5e1496d893d2fe121af70879ceadf